docs/rules/required-min-length-string-type-property.md

```diff
@@ -0,0 +1,90 @@
+# required-min-length-string-type-property
```

Comment:
I wonder if this rule should be renamed. 

`required-string-property-missing-min-length`

Reply:
I've renamed rule in this [PR](https://github.com/Redocly/redocly-cli/pull/1039)